src/box/field_default_func.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2023, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "port.h"
+#include "engine.h"
+#include "tt_static.h"
+#include "tuple_format.h"
+#include "field_default_func.h"
+
+/**
+ * Find `default_func` in func cache.
+ * Return NULL if something went wrong (diag is set).
+ */
+struct func *
+field_default_func_find(struct field_default_func *default_func)
```

Comment:
Nit: Should be static. Anyway, the function is so tiny and used only in two places so better drop it and inline its code instead.

Reply:
Done.